openslides_backend/action/generics.py

```diff
@@ -432,6 +432,12 @@ def delete_action_prepare_dataset(self, payload: ActionPayload) -> DataSet:
 
         return {"data": data}
 
+    def update_instances(self, payload: ActionPayload) -> ActionPayload:
```

Comment:
What is the difference to `get_updated_instances` which is defined in base.py? This new method is not used in this PR.

Reply:
Ah, probably an error from the rebase. Will fix once #245 is merged. Good catch

Reply:
removed.